### PR TITLE
Edge link redirecting

### PR DESCRIPTION
**Pull request by dianaolives**
Modify links directed towards Edge documentation so that they redirect to older Edge documentation.

### CODE REVIEW

File: content/glossary/c.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ For details see [Analytics > DataHub](/datahub/datahub-overview/).
 
 {{< product-c8y-iot >}} Edge is the onsite solution of {{< product-c8y-iot >}} intended to run as a local software application on industrial PC’s or local servers.
 
-For details see the [Edge](/edge/edge-introduction/).
+For details see the [Edge](https://cumulocity.com/guides/edge/edge-introduction/).
```
Comment (BeateRixen) [inline content/glossary/c.md:39]:
```suggestion
For details see [Edge](https://cumulocity.com/guides/edge/edge-introduction/).
```

File: content/glossary/c.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ For details see [Analytics > DataHub](/datahub/datahub-overview/).
 
 {{< product-c8y-iot >}} Edge is the onsite solution of {{< product-c8y-iot >}} intended to run as a local software application on industrial PC’s or local servers.
 
-For details see the [Edge](/edge/edge-introduction/).
```
Comment (BeateRixen) [inline content/glossary/c.md:39]: @TzviLipschitz In the glossary, we should describe both Edge variants, either in one entry or in separate entries. Can you please take care of this in a separate PR?

Comment (TzviLipschitz) [reply content/glossary/c.md:39]: @BeateRixen No problem, will create a ticket and make the required changes.

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@dianaolives Please also cherry-pick to 2024

**Pull request merged**